Subject: [PATCH] NuPlayer: relax meta check condition for streaming

But it prevents work for streaming due to lack of data.
Test: manual - play something with native player (OpenMAX AL)

diff --git a/media/libmediaplayerservice/nuplayer/NuPlayer.cpp b/media/libmediaplayerservice/nuplayer/NuPlayer.cpp
index 4e16fba..2fd266c 100644
--- a/media/libmediaplayerservice/nuplayer/NuPlayer.cpp
+++ b/media/libmediaplayerservice/nuplayer/NuPlayer.cpp
@@ -1382,7 +1382,7 @@
 
     sp<MetaData> audioMeta = mSource->getFormatMeta(true /* audio */);
     sp<MetaData> videoMeta = mSource->getFormatMeta(false /* audio */);
-    if (audioMeta == NULL && videoMeta == NULL) {
+    if ((audioMeta == NULL && videoMeta == NULL) && !(mSource->isStreaming())) {
         ALOGE("no metadata for either audio or video source");
         mSource->stop();
         mSourceStarted = false;
